src/addons/hwtoken/src/main/java/ee/ria/xroad/signer/tokenmanager/module/HardwareModuleWorker.java

```diff
@@ -118,22 +120,29 @@ protected void deinitializeModule() throws Exception {
         }
 
         log.info("Module '{}' got {} slots", module.getType(), slots.length);
+        for (int i = 0; i < slots.length; i++) {
+            log.info("Module '{}' Slot {} ID: {} (0x{})", module.getType(), i, slots[i].getSlotID(),
```

Comment:
Looks like debug (or trace) output. Should probably be removed or at least the logging level should be changed.